packages/@webex/internal-plugin-user/test/integration/spec/user.js

```diff
@@ -123,10 +264,11 @@ runs.forEach((run) => {
           .then((res) => {
             assert.isTrue(res.verificationEmailTriggered);
             assert.property(res, 'verifyEmailURL');
-            const {query} = url.parse(res.verifyEmailURL);
-            const token = querystring.parse(query).t;
 
-            return unauthWebex.internal.user.activate({email, verificationToken: token});
+            return unauthWebex.internal.user.activate({
+              id: res.id,
+              confirmationCode: res.verifyEmailURL,
+            });
```

Comment:
Has the method itself changed here to receive `confirmationCode` instead of `verificationToken`?

Reply:
@mkesavan13  There is a small change. Either we can only pass verificationToken or confirmationCode with id. This flow that we have here is similar to what we are doing in web-client.

Reply:
Cool. Thanks, Shreyas